### PR TITLE
Adjust translations for PrimeNG 17

### PR DESCRIPTION
**Pull request by marcschroeder**

### CODE REVIEW

**Comment by melloware** [comment]
@marcschroeder sorry can you only ADD items and not delete items. The items you have deleted are used by PrimeReact and PrimeFaces.

**Review by melloware — CHANGES_REQUESTED** [review]
see comments

File: en.json [hunk 1]
```diff
@@ -8,6 +8,10 @@
       "notEquals": "Not equals",
       "noFilter": "No Filter",
       "filter": "Filter",
+      "is": "Is",
+      "after": "After",
+      "before": "Before",
+      "isNot": "Is not",
```
Comment (melloware) [inline en.json:14]: I could be wrong but should't PrimeNg be using these..

```js
    "dateIs": "Date is",
      "dateIsNot": "Date is not",
      "dateBefore": "Date is before",
      "dateAfter": "Date is after",
```
?

**Review by melloware — CHANGES_REQUESTED** [review]

File: en.json (hunk 1 above)
Comment (marcschroeder) [reply en.json:14]: I followed https://github.com/primefaces/primeng/blob/master/src/app/components/api/translation.ts for the keys.

Comment (melloware) [reply en.json:14]: OK I looked at the source code and those 4 are not in any other lib PrimeFaces, PrimeVue or PrimeReact.

It seems they are just shortcuts to the real methods.

```ts
        is: (value: any, filter: any, filterLocale?: any): boolean => {
            return this.filters.equals(value, filter, filterLocale);
        },

        isNot: (value: any, filter: any, filterLocale?: any): boolean => {
            return this.filters.notEquals(value, filter, filterLocale);
        },

        before: (value: any, filter: any, filterLocale?: any): boolean => {
            return this.filters.lt(value, filter, filterLocale);
        },

        after: (value: any, filter: any, filterLocale?: any): boolean => {
            return this.filters.gt(value, filter, filterLocale);
        },
```

So I actually think these 4 are deprecated in PrimeNG and replaced with the 4 values that have translations so somewhere along the lines PrimeTek made all the libraries in line and left these 4 for backwards compatibility but they should probably be marked deprecated.

I just opened this PrimeNG ticket: https://github.com/primefaces/primeng/issues/14446

**Pull request closed without merging**